common/app/experiments/Experiments.scala

```diff
@@ -82,3 +83,12 @@ object BillboardsInMerchSlots
       sellByDate = LocalDate.of(2023, 2, 1),
       participationGroup = Perc0E,
     )
+
+object HeaderTopBarSearchCapi
+    extends Experiment(
+      name = "header-top-bar-search-capi",
+      description = "Adds CAPI search to the top nav",
+      owners = Seq(Owner.withGithub("@guardian/dotcom-platform")),
+      sellByDate = LocalDate.of(2023, 2, 1),
+      participationGroup = Perc0E,
```

Comment:
Opt-in only? There’s a switch as well, so it can always be turned off…

Reply:
I guess if you want to be able to test something in PROD on your machine that’s the only way. But I’d consider being bold and rolling out to 1% straight away. Considering this button is very rarely clicked at all…

Reply:
Thanks for reminding me that that is a useful way to think - I've updated this.